### ISSUE TITLE
Website is inaccessible

### ISSUE THREAD

**Issue by KaiserPhemi** [date-fns/date-fns#2687]
### Description
 
Hi,
I'm trying to check the documentation on calculating age from date of birth but it appears the website is inaccessible.

<img width="841" alt="Screenshot 2021-10-11 at 21 56 14" src="https://user-images.githubusercontent.com/26272628/136854759-a3f2453d-90df-4437-bdca-d9f07d9fa897.png">

Browser: Firefox

**Comment by tan75** [comment]
hi @KaiserPhemi 

the docs website is fully operational however sometimes it is being blocked by a VPN as it runs from Firebase.
If you can't access the docs website, please use our Github repo as all the docs are available via JSDocs annotations e.g.
[Add function](https://github.com/date-fns/date-fns/blob/master/src/add/index.ts#L8-L46)
Hope this helps!

**Comment by fturmel** [comment]
Since this is sticky, I'm linking the following issue for anyone having trouble reaching the docs in a corporate environment with aggressive network filtering: https://github.com/date-fns/date-fns.org/issues/209

TLDR; You need to allow traffic to both the `date-fns.org` and `firestore.googleapis.com` domains for the documentation website to work correctly.
